### PR TITLE
Improving the readme

### PR DESCRIPTION
**Pull request by monikasmolarek**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -202,7 +202,7 @@ Please note that each of the post services exposes it's own API (`--operator-add
 
 While Dagu is small and easy to use, it might not be the best choice for a production environment. This really depends on your needs. Larger systems like Apache Airflow or Prefect have more features and integrations. But, they are also more complex and need more resources and expertise to manage.
 
-You might also succeed with different type of tools like n8n, depending on your setup.
```
Comment (pigmej) [inline readme.md]: So "type of tools" != "types of tools" isn't it? :)

Comment (monikasmolarek) [reply readme.md]: sure!

**Comment by monikasmolarek** [comment]
@nj merge or close?

**Review by pigmej — APPROVED** [review]

**Comment by pigmej** [comment]
> @nj merge or close?

Merged :) sorry that it took that long :)

**Pull request merged**